python/ray/serve/batching.py

```diff
@@ -242,6 +248,25 @@ def __del__(self):
         self._handle_batch_task.cancel()
 
 
+def _validate_max_batch_size(max_batch_size):
+    if not isinstance(max_batch_size, int):
+        if isinstance(max_batch_size, float) and max_batch_size.is_integer():
+            max_batch_size = int(max_batch_size)
+        else:
+            raise TypeError("max_batch_size must be integer >= 1")
```

Comment:
Could you modify these error messages since `max_batch_size` can now be a `Callable` as well?